.github/workflows/macos.yml

```diff
@@ -166,7 +166,18 @@ jobs:
       - name: Build dmg (${{ matrix.build-type }})
         run: |
           cd build
-          hdiutil create -srcfolder 'Notes Better.app' -format UDZO -fs HFS+ -volname 'Notes Better' '${{ steps.vars.outputs.file_name }}'
+          npm install --global create-dmg
```

Comment:
I should probably set a version here just to avoid future problems, right?

`6.0.0` is the latest one: https://www.npmjs.com/package/create-dmg?activeTab=versions

Reply:
Can we just specify the major version? It may still be good to pick up bugfixes and compatible improvements.

Reply:
Good thinking. Done.